.travis.yml

```diff
@@ -1,15 +1,34 @@
-# Don't use any predefined language env
-# (see https://github.com/travis-ci/travis-ci/issues/4895#issuecomment-150703192)
+# We will use docker to set up out environment, so don't use any particular
+# language in Travis itself
 language: generic
 
+# Enable docker
 sudo: required
+services:
+    - docker
 
-services: docker
+# Disable automatic submodule fetching (it's done recursively)
+git:
+    submodules: false
+
+# Do a shallow submodule fetch
+before_install: git submodule update --init
 
 env:
-    - DIST=trusty
-    - DIST=xenial
+    global:
+        # Make sure beaver is in the PATH
+        - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+    matrix:
+        - DIST=trusty
+        - DIST=xenial
+
+install: beaver install
 
-install: ci/travis-install.sh
+script: beaver make all deb
 
-script: ci/travis-script.sh
+deploy:
+    provider: script
+    script: beaver bintray upload -d sociomantic-tsunami/dlang/ebtree deb/*.deb
```

Comment:
I just made this small change to upload the files to the `sociomantic-tsunami/dlang` bintray repo. Having one repo per project is a bit overkill, as for now we only use ebtree in dlang projects, that should be enough.

If there is some external need for an independent repo, we can create one in the future and deploy to both.